app/decorators/reviewable.rb

```diff
@@ -5,6 +5,15 @@ def class
     __getobj__.class
   end
 
+  def attributes=(attributes)
+    if attributes["reviewed"] == "1"
```

Comment:
Or change the UI so it passes in 0 on selecting the "needs review" option, and 1 when not selected